torchrl/objectives/a2c.py

```diff
@@ -33,6 +35,34 @@
 )
 
 
+class LossContainerBase:
+    """ContainerBase class loss tensorclass's."""
```

Comment:
That isn't very explicit. We should say what this class is about.

Also I think it should live in the `common.py` file.

Reply:
I'm also wondering if we should not just make the base a tensorclass and inherit from it without creating new tensorclasses?

Reply:
If I try to make the base a tensorclass getting below error.
```
**********************************************************************
File "/home/sandish/rl/torchrl/objectives/a2c.py", line 144, in a2c.A2CLoss
Failed example:
    loss(data)
Exception raised:
    Traceback (most recent call last):
      File "/home/sandish/.conda/envs/torch_rl/lib/python3.9/doctest.py", line 1334, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest a2c.A2CLoss[21]>", line 1, in <module>
        loss(data)
      File "/home/sandish/.conda/envs/torch_rl/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
        return self._call_impl(*args, **kwargs)
      File "/home/sandish/.conda/envs/torch_rl/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1561, in _call_impl
        result = forward_call(*args, **kwargs)
      File "/home/sandish/.conda/envs/torch_rl/lib/python3.9/site-packages/tensordict/_contextlib.py", line 126, in decorate_context
        return func(*args, **kwargs)
      File "/home/sandish/.conda/envs/torch_rl/lib/python3.9/site-packages/tensordict/nn/common.py", line 291, in wrapper
        return func(_self, tensordict, *args, **kwargs)
      File "/home/sandish/rl/torchrl/objectives/a2c.py", line 503, in forward
        return A2CLosses._from_tensordict(td_out)
      File "/home/sandish/.conda/envs/torch_rl/lib/python3.9/site-packages/tensordict/tensorclass.py", line 327, in wrapper
        raise ValueError(
    ValueError: Keys from the tensordict ({'loss_entropy', 'loss_objective', 'entropy', 'loss_critic'}) must correspond to the class attributes (set()).
```